src/c++/perf_analyzer/genai-perf/genai_perf/profile_data_parser/profile_data_parser.py

```diff
@@ -26,13 +26,22 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from enum import Enum, auto
 from pathlib import Path
 from typing import List, Tuple
 
-from genai_perf.metrics import Metrics, ResponseFormat, Statistics
+from genai_perf.metrics import Metrics, Statistics
 from genai_perf.utils import load_json
 
 
+class ResponseFormat(Enum):
```

Comment:
Moved over from `metrics.py` since the enum is more relevant to the data parser classes.

Reply:
Is this duplicated in two places? This is also in the LLMInputs class, I think. We need to fix that.

Reply:
They are slightly different - one's used for input to PA and another for parsing PA output. There's no notion of "backend" when parsing profile export data, so `ResponseFormat` has `TRITON` instead of `TENSORRTLLM` and `VLLM` as in `OutputFormat`. I think part of the confusion is coming from the name of the class `OutputFormat` because I think it should really be called `InputFormat` 😅

Reply:
Thanks for explaining! Agreed, I like the idea of updating the naming that way (outside of this PR).

Reply:
Oh, they already have different names. Missed that last night. 😓